tests/test_cli.py

```diff
@@ -37,6 +37,25 @@ def test_list(cli, dotenv_path, format: Optional[str], content: str, expected: s
     assert (result.exit_code, result.output) == (0, expected)
 
 
+@pytest.mark.parametrize(
+    "contents,expected",
+    (
+        (["x='1'", "y='2'"], '''x=1\ny=2\n'''),
+        (["x='1'", "x='2'"], '''x=2\n'''),
+        (["x='1'\ny='2'", "y='20'\nz='30'"], '''x=1\ny=20\nz=30\n'''),
+    )
+)
+def test_list_multi_file(cli, dotenv_path, extra_dotenv_path, contents: List[str], expected: str):
+    dotenv_path.write_text(contents[0] + '\n')
+    extra_dotenv_path.write_text(contents[1] + '\n')
+
+    args = ['--file', dotenv_path, '--file', extra_dotenv_path, 'list']
```

Comment:
I omitted tests with multiple formatting because that seems to me like a separate concern - if the variables are loaded correctly, the formatting behavior shouldn't change.

Do you agree?

Reply:
You are right. This is fine.